website/project/model.py

```diff
@@ -3580,6 +3629,17 @@ def _on_reject(self, user, token):
         )
 
 
+class DraftRegistrationApproval(Sanction):
+
+    mode = Sanction.ANY
+
+    def _on_complete(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
+
+    def _on_reject(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
+
+
 class DraftRegistration(AddonModelMixin, StoredObject):
 
     is_draft_registration = True
```

Comment:
To a reader unfamiliar with this class, this field name might seem confusing. Can you provide a short comment describing when a draft registration is [no longer] a draft registration?